project/Mima.scala

```diff
@@ -17,7 +17,17 @@ object Mima {
       ProblemFilters.exclude[ReversedMissingMethodProblem]("scalafix.interfaces.ScalafixFileEvaluation.getErrorMessage"),
       ProblemFilters.exclude[ReversedMissingMethodProblem]("scalafix.interfaces.ScalafixEvaluation.getError"),
       ProblemFilters.exclude[ReversedMissingMethodProblem]("scalafix.interfaces.ScalafixEvaluation.getErrorMessage"),
-      ProblemFilters.exclude[ReversedMissingMethodProblem]("scalafix.interfaces.ScalafixFileEvaluation.getError")
+      ProblemFilters.exclude[ReversedMissingMethodProblem]("scalafix.interfaces.ScalafixFileEvaluation.getError"),
+      ProblemFilters.exclude[MissingTypesProblem]("scala.meta.internal.pc.ScalafixGlobal"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.internal.pc.ScalafixGlobal.presentationCompilerThread"),
+      ProblemFilters.exclude[MissingClassProblem]("scala.tools.nsc.interactive.ScalafixGlobalProxy"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.internal.pc.ScalafixGlobal.hijackPresentationCompilerThread"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.internal.pc.ScalafixGlobal.isHijacked"),
+      ProblemFilters.exclude[DirectMissingMethodProblem]("scala.meta.internal.pc.ScalafixGlobal.metalsAsk"),
+      ProblemFilters.exclude[MissingClassProblem]("scala.tools.nsc.interactive.ScalafixGlobalThread"),
+      ProblemFilters.exclude[MissingClassProblem]("scala.tools.nsc.interactive.ScalafixGlobalThread$"),
+      ProblemFilters.exclude[MissingClassProblem]("scala.tools.nsc.interactive.InterruptException"),
+      ProblemFilters.exclude[MissingClassProblem]("scala.tools.nsc.interactive.InterruptException$")
```

Comment:
Maybe classes that are outside the scalafix namespace just to access internals should be fully excluded?
```suggestion
      ProblemFilters.exclude[MissingTypesProblem]("scala.meta.internal.*"),
      ProblemFilters.exclude[MissingClassProblem]("scala.tools.nsc.interactive.*")
```

Reply:
better, in fact.